suite-common/firmware/src/firmwareReducer.ts

```diff
@@ -101,6 +104,20 @@ export const prepareFirmwareReducer = createReducerWithExtraDeps(initialState, (
                 if (!(action.type === DEVICE.BUTTON && state.status === 'initial'))
                     state.uiEvent = action;
             },
+        )
+        .addMatcher(
+            isAnyOf(deviceActions.connectDevice, deviceActions.connectUnacquiredDevice),
```

Comment:
I copied this from [deviceReducer](https://github.com/trezor/trezor-suite/blob/b870aa9ff159888c2d813cd5d1aa318b7f7d9f4d/suite-common/wallet-core/src/device/deviceReducer.ts#L648), but maybe `connectUnacquiredDevice` is not relevant? I'm not 100% what it means to "acquire" a device..